components/automate-ui/src/app/modules/team/team-add-users/team-add-users.component.html

```diff
@@ -12,12 +12,17 @@
   <div *ngIf="!(loading$ | async)">
     <app-create-user-modal [openEvent]="openUserModal"></app-create-user-modal>
     <div id="no-users-container" *ngIf="usersNotFiltered().length === 0">
-      <p>All local users have already been added; create some more!</p>
-      <chef-button primary (click)="openModal()">Create User</chef-button>
+      <app-authorized not [allOf]="['/iam/v2/users', 'post']">
+        <p>All local users have already been added.</p>
+      </app-authorized>
+      <app-authorized [allOf]="['/iam/v2/users', 'post']">
+        <p>All local users have already been added; create some more!</p>
+        <chef-button primary (click)="openModal()">Create User</chef-button>
+      </app-authorized>
     </div>
     <div id="table-container" *ngIf="usersNotFiltered()?.length > 0">
       <div id="table-controls">
-        <app-authorized [anyOf]="['/iam/v2/teams/{id}/users:add', 'post', team?.id]">
+        <app-authorized [allOf]="['/iam/v2/users', 'post']">
```

Comment:
While I agree that having the users permission there is good, I would suggest also include the original teams users:add as well, because that is not being checked prior to getting here. (It should be, but that looks like a bit larger can of worms to fix that.)

Reply:
I switched it because the app-authorized is only hiding the create button. 
Started looking into how we could start guarding the `Add Users` buttons on the team-details page and realized it would take some refactoring of the app-user-table. I'll make an issue for that fix.

Reply:
issue: https://github.com/chef/automate/issues/2762